.gitlab/build_lassen.yml

```diff
@@ -0,0 +1,45 @@
+####
+# This is the share configuration of jobs for lassen
+.on_lassen:
+  variables:
+  tags:
+    - shell
+    - lassen
+  rules:
+    - if: '$CI_COMMIT_BRANCH =~ /_lnone/ || $ON_LASSEN == "OFF"' #run except if ...
+      when: never
+    - when: on_success
+
+####
+# Load required CUDA module
+.with_cuda:
+  before_script:
+    - module load cuda/10.1.243
+
+####
+# Template
+.pr_build_on_lassen:
+  stage: l_build
+  extends: [.build_blueos_3_ppc64le_ib_p9_script, .on_lassen, .pr_workflow]
+  needs: []
+
+.main_build_with_deps_on_lassen:
+  stage: l_build_with_deps
+  extends: [.build_with_deps_blueos_3_ppc64le_ib_p9_script, .on_lassen, .main_workflow]
+  needs: []
+
+####
+# Build jobs
```

Comment:
Looks like future PRs will add other host-configs?

Reply:
I'll do it now they should just work